modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,17 @@
+[[hostname-rename]]
+= Rename Hostname
+
+With the [command]``spacewalk-hostname-rename`` script it is possible to rename the hostname of the {productname} Server.
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+It is the best to proceed as follows: 
+
+. Adjust the hostname in advance.
+It is important to configure it locally, and for DNS resolution (including reverse resolution).
+. Load the new configuration.
+If in doubt whether the new configuration is running, reboot the server.
```

Comment:
Wouldn't it be easier if we just say "Reboot the server" without conditionals?

Reply:
if done right, rebooting is not needed.  I do not like recommending a reboot if it is not strictly necessary.  Nevertheless I'll change the text.  If you (one of you) still disagree, I can simplify it even more.